### PR TITLE
adding desk prompt

### PR DESCRIPTION
**Pull request by remche**
A prompt segment for desk shell environment.
https://github.com/jamesob/desk

### CODE REVIEW

File: fish_prompt.fish [hunk 1]
```diff
@@ -927,11 +934,15 @@ function __bobthefish_maybe_display_colors -S
   echo -ns $__bobthefish_virtualenv_glyph virtualfish ' '
   __bobthefish_finish_segments
 
-  __bobthefish_start_segment $__color_virtualgo
+   __bobthefish_start_segment $__color_virtualgo
```
Comment (bobthecow) [inline fish_prompt.fish]: something happened with whitespace here?

File: fish_prompt.fish [hunk 2]
```diff
@@ -1132,6 +1148,7 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
       set __color_rvm                      brmagenta $colorfg --bold
       set __color_virtualfish              brblue $colorfg --bold
       set __color_virtualgo                brblue $colorfg --bold
+      set __color_desk                brblue $colorfg --bold
```
Comment (bobthecow) [inline fish_prompt.fish]: whitespace

File: fish_prompt.fish [hunk 3]
```diff
@@ -1409,6 +1431,7 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
       set __color_rvm                      $red $colorfg --bold
       set __color_virtualfish              $cyan $colorfg --bold
       set __color_virtualgo                $cyan $colorfg --bold
+      set __color_desk                $cyan $colorfg --bold
```
Comment (bobthecow) [inline fish_prompt.fish]: whitespace

**Comment by remche** [comment]
Makes sense, I totally agree that silly copy-paste is a poor solution... I'm not a fish expert, and I just commited what I did for myself.

Could we use a fish function to generate the segment ?

**Comment by bobthecow** [comment]
yeah, we could. i gave it a shot, but there's enough different about each of the instances of this that it was hard to come up with something completely declarative. i won't hold up this request any longer for that :)

**Pull request merged**